CHANGELOG.md

```diff
@@ -1,8 +1,8 @@
 # Changelog
 
-## [cryptoki-sys-0.1.4](https://github.com/parallaxsecond/rust-cryptoki/tree/cryptoki-sys-0.1.4) (2022-08-03)
+## [cryptoki-0.4.0](https://github.com/parallaxsecond/rust-cryptoki/tree/cryptoki-0.4.0) (2022-09-07)
```

Comment:
Should there be a breaking changes section to highlight in what way this release is not backward compatible with the last release?